src/firebaseConfig.env.ts

```diff
@@ -0,0 +1,9 @@
+export const firebaseConfig = {
+    apiKey: 'AIzaSyAPBd1RS5KKXLmvBSVHB0e3GIbJHwX_Nlc',
+    authDomain: 'weatherapp-9ffa0.firebaseapp.com',
+    projectId: 'weatherapp-9ffa0',
+    storageBucket: 'weatherapp-9ffa0.appspot.com',
+    messagingSenderId: '7546937149',
+    appId: '1:7546937149:web:ecbfff8602e77de7ea6f54',
+    measurementId: 'G-RE5ML33910'
```

Comment:
create a .env file and move all of magic string into it. each variable must have name like REACT_APP_SOME_VAR and then access them like process.env.REACT_APP_SOME_VAR